src/style.scss

```diff
@@ -0,0 +1,186 @@
+/* #region root */
+$main-accent: #060b35;
+$secondary: #616070;
+$blue-accent: #00acdc;
+$white: #fff;
+
+:root {
+  --main-accent: #{$main-accent};
+  --secondary: #{$secondary};
+  --blue-accent: #{$blue-accent};
+  --white: #{$white};
+}
+
+body {
+  margin: 0;
+  display: flex;
+}
+
+/* #endregion */
+
+/* #region card */
+
+.card {
+  width: 198px;
+  height: 406px;
+  display: flex;
+
+  flex-direction: column;
+
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin: auto 19px;
+    margin-top: 32px;
+    margin-bottom: 0;
+  }
+
+  &__span-1 {
+    width: 166px;
+
+    color: var(--main-accent);
+    font-size: 12px;
+    font-family: Roboto, sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 18px;
+
+    margin: auto 16px;
+    margin-top: 40px;
+    margin-bottom: 4px;
+  }
+
+  &__span-2 {
+    color: var(--secondary);
+    font-size: 10px;
+    font-family: Roboto, sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px;
+
+    margin: 0 auto;
+    margin-left: 16px;
+    margin-top: 0;
+  }
+}
+
+/* #endregion */
+
+/* #region stars */
+
+.card__stars-div {
+  display: flex;
+  justify-content: space-between;
+
+  margin: auto 16px;
+
+  margin-top: 16px;
+}
+
+.card__stars {
+  display: flex;
+}
+
+.card__star {
```

Comment:
please use only 2 BEM blocks - 1 for stars, 1 for card, both of them should have their own files